packages/blockchain-wallet-v4/src/exchange/currencies/USDD.ts

```diff
@@ -0,0 +1,26 @@
+export default {
+  base: 'WEI',
+  code: 'PAX',
+  displayName: 'USD Pax',
+  trade: 'PAX',
+  units: {
+    WEI: {
+      rate: '1',
+      symbol: 'WEI',
+      decimal_digits: 0,
+      currency: 'PAX'
+    },
+    GWEI: {
+      rate: '1000000000',
+      symbol: 'GWEI',
+      decimal_digits: 0,
+      currency: 'PAX'
+    },
+    PAX: {
+      rate: '1000000000000000000',
+      symbol: 'USD-D',
+      decimal_digits: 2,
+      currency: 'PAX'
+    }
+  }
+}
```

Comment:
Not sure about this one. PAX is the currency, USD-D is the formatted currency name. We already have all that info via supportedCoins and wallet options. This would add another config for PAX

Reply:
Changes made to branch. Also includes the minimumwithdrawl fix for earn interest.